editor/plugins/script_text_editor.cpp

```diff
@@ -81,7 +81,7 @@ bool ScriptTextEditor::goto_method(const String &p_method) {
 		if (function.begins_with(method_search)) {
 
 			int line = function.get_slice(":", 1).to_int();
-			goto_line(line - 1);
+			call_deferred("_goto_line", line - 1);
```

Comment:
This is needed if the ScriptTextEditor was just created, to give the containers time to resize the TextEdit properly. Otherwise it will scroll past end of file (even if the option is disabled).